sonar-scala-plugin/src/main/resources/org/sonar/l10n/scala/rules/scala/S1313.html

```diff
@@ -0,0 +1,39 @@
+<p>Hardcoding IP addresses is security-sensitive. It has led in the past to the following vulnerabilities:</p>
+<ul>
+  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2006-5901">CVE-2006-5901</a> </li>
+  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-3725">CVE-2005-3725</a> </li>
+</ul>
+<p>Today's services have an ever-changing architecture due to their scaling and redundancy needs. It is a mistake to think that a service will always
+have the same IP address. When it does change, the hardcoded IP will have to be modified too. This will have an impact on the product development,
+delivery and deployment:</p>
+<ul>
+  <li> The developers will have to do a rapid fix every time this happens, instead of having an operation team change a configuration file. </li>
+  <li> It forces the same address to be used in every environment (dev, sys, qa, prod). </li>
+</ul>
+<p>Last but not least it has an effect on application security. Attackers might be able to decompile the code and thereby discover a potentially
+sensitive address. They can perform a Denial of Service attack on the service at this address or spoof the IP address. Such an attack is always
+possible, but in the case of a hardcoded IP address the fix will be much slower, which will increase an attack's impact.</p>
+<h2>Recommended Secure Coding Practices</h2>
+<ul>
+  <li> make the IP address configurable. </li>
+</ul>
+<h2>Noncompliant Code Example</h2>
+<pre>
+val ip = "192.168.12.42" // Noncompliant
+val socket new Socket(ip, 6667))
```

Comment:
I think you forgot to add `=` and there is an extra parenthesis at the end.

Reply:
Yes of course, the code should be: val socket = new Socket(ip, 6667)